draft-ietf-quic-transport.md

```diff
@@ -2389,11 +2389,17 @@ of bytes following it that are set to unpredictable values.  The last 16 bytes
 of the datagram contain a Stateless Reset Token.
 
 To entities other than its intended recipient, a stateless reset will appear
-to be a packet with a short header.  For the packet to appear as valid, the
-Unpredictable Bits field needs to include at least 198 bits of data (or 25
-bytes, less the two fixed bits).  This is intended to allow for a Destination
-Connection ID of the maximum length permitted, with a minimal packet number, and
-payload.  The Stateless Reset Token corresponds to the minimum expansion of the
+to be a packet with a short header.  For the stateless reset to appear as a
+valid QUIC packet and be smaller than the received packet, the Unpredictable
```

Comment:
Needing "at least 48 bits" isn't a requirement to "be smaller than the received packet."  This addition doesn't compute.

Reply:
What Mike said.  I think that you want to say:

> A stateless reset MUST include an Unpredictable Bits field of at least 38 bits (or 5 bytes, less the two fixed bits).  However, this does not account for the length of the connection ID chosen by a peer, which is not known by the endpoint sending a stateless reset.  For a stateless reset to appear to be a valid QUIC packet, the packet needs to appear to contain one type byte, a minimum-sized connection ID, four bytes of packet number and payload, and the AEAD expansion. In order to ensure that a stateless reset is smaller than the packet that triggers it, an endpoint needs to send a packet that is one byte longer than the minimum stateless reset size.  Therefore, endpoints SHOULD always send packets that are at least 22 bytes larger than the minimum connection ID that they might select.
>
>These calculations assume a 16 byte AEAD expansion.  Smaller packets can be sent if the AEAD expansion is less than 16 bytes and larger packets need to be sent for a larger expansion.